bits.go

```diff
@@ -266,3 +267,69 @@ func fromDeltas(data []byte, buf []uint32) []uint32 {
 	}
 	return buf
 }
+
+type runeOffset struct {
+	x, y uint32
```

Comment:
can you give these better variable names. x and y do not explain them. I believe `x` is a `runeOffset` and `y` is the `byteOffset` for `x`. Is that correct?